### PR TITLE
Add AllCops exclusions

### PR DESCRIPTION
**Pull request by wuj818**
This PR adds some exclusions to `AllCops`.

### CODE REVIEW

File: dev_tools/rubocop/base.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+AllCops:
+  Exclude:
+    - 'bin/{rails,rake}'
+    - 'db/**/*'
```
Comment (isaacdudek) [inline dev_tools/rubocop/base.yml]: Does this exclude `db/seeds.rb`? Sometimes that can get quite complex so I'm not sure we'd want to exclude it.

**Review by isaacdudek — CHANGES_REQUESTED** [review]

File: dev_tools/rubocop/base.yml (hunk 1 above)
Comment (wuj818) [reply dev_tools/rubocop/base.yml]: I changed it to be more explicit.

**Comment by justine-ga** [comment]
Is there a ticket for this PR? I don't really understand the context.

**Comment by wuj818** [comment]
There's no related ticket, this PR just excludes some unnecessary files from Rubocop.

I also added a Readme.

**Review by isaacdudek — APPROVED** [review]

**Pull request merged**